plugins/techdocs-backend/src/techdocs/stages/prepare/github.test.ts

```diff
@@ -16,11 +16,13 @@
 
 import { getVoidLogger } from '@backstage/backend-common';
 import { GithubPreparer } from './github';
-import { checkoutGitRepository } from './helpers';
+import { checkoutGithubRepository } from './helpers';
 
 jest.mock('./helpers', () => ({
   ...jest.requireActual<{}>('./helpers'),
-  checkoutGitRepository: jest.fn(() => '/tmp/backstage-repo/org/name/branch'),
+  checkoutGithubRepository: jest.fn(
+    () => '/tmp/backstage-repo/org/name/branch',
```

Comment:
I can see that you didn't add the `/tmp` but I am curious if this is a good practice. We might want to use a temp dir library for cross platform support. 😅 

@spotify/backstage-core Do you have any inputs on this? 🙇

Reply:
If it actually accesses the file system in the tests (haven't read all the code), then yeah we should probably use a proper tmpdir thing and carefully clean up.

Reply:
Cool, thanks @freben!
@JacobValdemar I think, no action is required from you here. :)